src/Microsoft.Framework.PackageManager/Commands/WrapCommand.cs

```diff
@@ -30,7 +30,7 @@ public class WrapCommand
         public string Framework { get; set; }
         public Reports Reports { get; set; }
 
-        public bool ExecuteCommand()
+        public bool Execute()
```

Comment:
We can keep this. In previous approach, @BrennanConroy 's changes conflict with mine so I asked him to make this change. All other `XYZCommand.ExecuteCommand()` has been renamed to `XYZCommand.Execute()` by me. This is the only one left.